packages/driver/src/fetchConn.ts

```diff
@@ -133,6 +133,8 @@ class BaseFetchConnection extends BaseRawConnection {
       this.messageWaiter.set();
     } catch (e) {
       this.messageWaiter.setError(e);
+    } finally {
+      this.messageWaiter = null;
```

Comment:
Moved clearing the `messageWaiter` into a `finally` block so that it's definitely and for sure non-null in the catch block. Does that work ok for you?

Reply:
Yes, this looks good to me.